Gems/Atom/Feature/Common/Code/Source/AuxGeom/AuxGeomDrawQueue.cpp

```diff
@@ -584,6 +585,176 @@ namespace AZ
+                            NearTopLeft, FarTopRight, NearTopRight,
+                        };
+
+                        Color transparentColor(color.GetR(), color.GetG(), color.GetB(), color.GetA() * 0.3f);
+                        drawArgs.m_indices = triangleIndices;
+                        drawArgs.m_indexCount = 36;
+                        drawArgs.m_colors = &transparentColor;
+                        drawArgs.m_opacityType = OpacityType::Translucent;
+                        DrawTriangles(drawArgs, faceCull);
+                    }
+                }
+
+                if (drawNormals)
+                {
+                    Vector3 planeNormals[] =
+                    {
+                        //near
+                        0.25f * (corners[NearBottomLeft] + corners[NearBottomRight] + corners[NearTopLeft] + corners[NearTopRight]),
+                        0.25f * (corners[NearBottomLeft] + corners[NearBottomRight] + corners[NearTopLeft] + corners[NearTopRight]) + frustum.GetPlane(Frustum::PlaneId::Near).GetNormal(),
+
+                        //far
+                        0.25f * (corners[FarBottomLeft] + corners[FarBottomRight] + corners[FarTopLeft] + corners[FarTopRight]),
+                        0.25f * (corners[FarBottomLeft] + corners[FarBottomRight] + corners[FarTopLeft] + corners[FarTopRight]) + frustum.GetPlane(Frustum::PlaneId::Far).GetNormal(),
+
+                        //left
+                        0.5f * (corners[NearBottomLeft] + corners[NearTopLeft]),
+                        0.5f * (corners[NearBottomLeft] + corners[NearTopLeft]) + frustum.GetPlane(Frustum::PlaneId::Left).GetNormal(),
+
+                        //right
+                        0.5f * (corners[NearBottomRight] + corners[NearTopRight]),
+                        0.5f * (corners[NearBottomRight] + corners[NearTopRight]) + frustum.GetPlane(Frustum::PlaneId::Right).GetNormal(),
+
+                        //bottom
+                        0.5f * (corners[NearBottomLeft] + corners[NearBottomRight]),
+                        0.5f * (corners[NearBottomLeft] + corners[NearBottomRight]) + frustum.GetPlane(Frustum::PlaneId::Bottom).GetNormal(),
+
+                        //top
+                        0.5f * (corners[NearTopLeft] + corners[NearTopRight]),
+                        0.5f * (corners[NearTopLeft] + corners[NearTopRight]) + frustum.GetPlane(Frustum::PlaneId::Top).GetNormal(),
+                    };
+
+                    Color planeNormalColors[] =
+                    {
+                        Colors::Red, Colors::Red,       //near
+                        Colors::Green, Colors::Green,   //far
+                        Colors::Blue, Colors::Blue,     //left
+                        Colors::Orange, Colors::Orange, //right
+                        Colors::Pink, Colors::Pink,     //bottom
+                        Colors::MediumPurple, Colors::MediumPurple, //top
+                    };
```

Comment:
Doing this kills the alpha information from the color argument.

Reply:
Not sure what alpha information you're referring to. The normals draw using their own colors, they don't care about the input color. (for better or worse - it's how the culling system did it and seemed fine to help distinguish which normal you're looking at).